src/libs/ReportUtils.ts

```diff
@@ -3176,15 +3189,27 @@ function addDomainToShortMention(mention: string): string | undefined {
  * For comments shorter than or equal to 10k chars, convert the comment from MD into HTML because that's how it is stored in the database
  * For longer comments, skip parsing, but still escape the text, and display plaintext for performance reasons. It takes over 40s to parse a 100k long string!!
  */
-function getParsedComment(text: string, shouldEscapeText?: boolean): string {
+function getParsedComment(text: string, parsingDetails?: ParsingDetails): string {
+    let isGroupPolicyReport = false;
+    if (parsingDetails?.reportID) {
+        const currentReport = getReport(parsingDetails?.reportID);
+        isGroupPolicyReport = isReportInGroupPolicy(currentReport);
+    }
+    if (parsingDetails?.policyID) {
```

Comment:
Oh this is interesting, but I don't think we should be supporting report mentions in any other place besides comments in reports, for two reasons, one we don't replace them in the backed anywhere else so they will be forever a name based mention, and two the mention is highlighted in the room description but you can't click on it to go to the room.

Thoughts?

Reply:
Yes, I think name based mention should be for optimistic data only before backend replace it with reportID based mention, because name based mention will stop navigate to the room if the room name changed.

Other than comments, we send report mentions in room description when room create/description edited and also when workspace description edited.

Reply:
I think we can easily do that by stopping pass `policyID` parameter in `getParsedComment`, and remove it if there is no usage left for it.
cc @war-in.

Reply:
Done :)